knexfile.js

```diff
@@ -0,0 +1,26 @@
+// Update with your config settings.
+
+module.exports = {
```

Comment:
I also set up our .knexfile on my branch, so just a heads up that we will likely have some merge conflicts.